src/data/lib/prim/Agda/Builtin/Reflection/External.agda

```diff
@@ -0,0 +1,14 @@
+{-# OPTIONS --without-K --safe --no-sized-types --no-guardedness --no-subtyping #-}
+
+module Agda.Builtin.Reflection.External where
+
+open import Agda.Builtin.List
+open import Agda.Builtin.Nat
+open import Agda.Builtin.Sigma
+open import Agda.Builtin.String
+
+postulate
+  execTC : String → List String → String
```

Comment:
To document the purpose of each of these string arguments and outputs, I suggest to use one or a combination of these methods:
- type synonyms for `String`
- named arguments
- documentation in comment

Reply:
I feel that adding type synonyms to a builtin module might be bloating it a little bit. Instead, I'll use named arguments, and a comment explaining the arguments and the return time.

Reply:
Ok, go ahead!